pkg/preparer/podprocess/finish_exec.go

```diff
@@ -0,0 +1,13 @@
+package podprocess
```

Comment:
I don't think so, are you suggesting I merge it into `reporter.go`? The reason I had it separate is because the preparer calls this function directly to determine the finish exec script to write, so it seemed separate from `reporter.go` in terms of purpose